### ISSUE TITLE
Can't Require prettydiff in v1.16.26

### ISSUE THREAD

**Issue by barakyo** [prettydiff/prettydiff#282]
I'm getting the error:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'prettydiff'
```

When trying to require prettydiff with the latest version.

**Comment by prettydiff** [comment]
Away from a computer right now. I will republish within the hour.  Is the prettydiff.js file missing?

**Comment by mattalat** [comment]
It looks like `prettydiff.js` is indeed missing

**Comment by prettydiff** [comment]
This is a massive NPM bug where the file indicated as the `index` in the package.json file is not included in the package tarball.  I am bumping the version to .27 and republishing.

**Comment by prettydiff** [comment]
New version published, v1.16.27 and I verified that prettydiff.js is present in the package.
